### ISSUE TITLE
Validate and QA the POC cluster

### ISSUE THREAD

**Issue by mgreensmith** [mozilla-itcloud/mdn-migration-project#11]

**Comment by mgreensmith** [comment]
Definition of done: end user can load the page, all primary links work, search works.

**Comment by mgreensmith** [comment]
Depends on #10

**Comment by escattone** [comment]
This is done. I can load the main page (https://developer.mozilla-mdn.nubis.allizom.org/en-US/) and all of its primary links work, and search works. I also configured the DB for OAuth, enabled a waffle flag, and then tested a few more things. First, login via GitHub works. Second, I ran the headless tests. The four failures are all expected due to the missing legacy files and file attachments on the EFS instance.
```sh
(mdntest) rjohnson-25186:kuma rjohnson$ pytest --base-url https://developer.mozilla-mdn.nubis.allizom.org tests/headless
=============================================== test session starts ===============================================
platform darwin -- Python 2.7.12, pytest-3.5.1, py-1.5.3, pluggy-0.6.0
sensitiveurl: .* *** WARNING: sensitive url matches https://developer.mozilla-mdn.nubis.allizom.org ***
metadata: {'Python': '2.7.12', 'Driver': None, 'Capabilities': {}, 'Base URL': 'https://developer.mozilla-mdn.nubis.allizom.org', 'Platform': 'Darwin-17.5.0-x86_64-i386-64bit', 'kuma': {u'services': {u'search': {u'available': True, u'count': 939, u'populated': True}, u'kumascript': {u'available': True, u'revision': u'79334de03084e208777d9a3923d786079d5d289e'}, u'test_accounts': {u'available': True}, u'database': {u'available': True, u'document_count': 957, u'populated': True}}, u'version': 1, u'request': {u'url': u'https://developer.mozilla-mdn.nubis.allizom.org/_kuma_status.json', u'is_secure': True, u'host': u'developer.mozilla-mdn.nubis.allizom.org', u'scheme': u'https'}, 'response': {'headers': {'X-Cache': 'Miss from cloudfront', 'strict-transport-security': 'max-age=63072000', 'x-content-type-options': 'nosniff', 'Content-Language': 'en-US', 'Content-Encoding': 'gzip', 'Transfer-Encoding': 'chunked', 'Expires': 'Thu, 21 Jun 2018 00:08:32 GMT', 'Vary': 'Accept-Encoding', 'Server': 'meinheld/0.6.1', 'Connection': 'keep-alive', 'x-xss-protection': '1; mode=block', 'X-Amz-Cf-Id': 'uO5XtCW61ZmVbRhawMvWi5O16W4R5dKjvqWYq2_Uvx8Abddu5FytiQ==', 'Cache-Control': 'no-cache, no-store, must-revalidate, max-age=0', 'Date': 'Thu, 21 Jun 2018 00:08:32 GMT', 'X-Frame-Options': 'DENY', 'Content-Type': 'application/json', 'Via': '1.1 323815e0c8825597e430b3273b4de2d0.cloudfront.net (CloudFront)'}}, u'settings': {u'ATTACHMENT_HOST': u'files.mozilla-mdn.nubis.allizom.org', u'PROTOCOL': u'https://', u'INTERACTIVE_EXAMPLES_BASE': u'https://interactive-examples.mdn.mozilla.net', u'MAINTENANCE_MODE': False, u'STATIC_URL': u'https://developer.mozilla-mdn.nubis.allizom.org/static/', u'SITE_URL': u'https://developer.mozilla-mdn.nubis.allizom.org', u'ATTACHMENT_ORIGIN': u'files.mozilla-mdn.nubis.allizom.org', u'DEBUG': False, u'ALLOWED_HOSTS': [u'developer.mozilla-mdn.nubis.allizom.org', u'stage.mozilla-mdn.nubis.allizom.org', u'files.mozilla-mdn.nubis.allizom.org'], u'REVISION_HASH': u'82e1ab8e71b9d3e3fc18dbd63ced0de3433209f0', u'LEGACY_HOSTS': []}}, 'Plugins': {'variables': '1.7.1', 'selenium': '1.11.4', 'xdist': '1.16.0', 'rerunfailures': '2.1.0', 'html': '1.16.1', 'base-url': '1.4.1', 'metadata': '1.5.1'}, 'Packages': {'py': '1.5.3', 'pytest': '3.5.1', 'pluggy': '0.6.0'}}
baseurl: https://developer.mozilla-mdn.nubis.allizom.org
rootdir: /Users/rjohnson/repos/kuma, inifile: pytest.ini
plugins: xdist-1.16.0, variables-1.7.1, selenium-1.11.4, rerunfailures-2.1.0, metadata-1.5.1, html-1.16.1, base-url-1.4.1
collected 530 items

tests/headless/test_cdn.py ................................................FF............................FF....s....................................................................................................................................................................................................................................................................... [  0%]
tests/headless/test_endpoints.py ...                                                                        [  0%]
tests/headless/test_redirects.py ................................................................................................................................................................................ [  0%]
tests/headless/test_robots.py ...

==================================================== FAILURES =====================================================
__________________________________ test_cached[/diagrams/workflow/workflow.svg] ___________________________________
Traceback (most recent call last):
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 254, in test_cached
    assert_cached(base_url + slug, 200, is_behind_cdn)
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 84, in assert_cached
    assert response.status_code == expected_status_code
AssertionError: assert 404 == 200
 +  where 404 = <Response [404]>.status_code
______________________________ test_cached[/presentations/microsummaries/index.html] ______________________________
Traceback (most recent call last):
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 254, in test_cached
    assert_cached(base_url + slug, 200, is_behind_cdn)
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 84, in assert_cached
    assert response.status_code == expected_status_code
AssertionError: assert 404 == 200
 +  where 404 = <Response [404]>.status_code
______________________________________ test_cached_301[/files/2767/hut.jpg] _______________________________________
Traceback (most recent call last):
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 267, in test_cached_301
    assert_cached(base_url + slug, 301, is_behind_cdn)
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 84, in assert_cached
    assert response.status_code == expected_status_code
AssertionError: assert 404 == 301
 +  where 404 = <Response [404]>.status_code
_________________________________ test_cached_301[/@api/deki/files/3613/=hut.jpg] _________________________________
Traceback (most recent call last):
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 267, in test_cached_301
    assert_cached(base_url + slug, 301, is_behind_cdn)
  File "/Users/rjohnson/repos/kuma/tests/headless/test_cdn.py", line 84, in assert_cached
    assert response.status_code == expected_status_code
AssertionError: assert 404 == 301
 +  where 404 = <Response [404]>.status_code
============================================= short test summary info =============================================
SKIP [1] tests/headless/test_cdn.py:297: unconditional skip
================================ 4 failed, 525 passed, 1 skipped in 136.09 seconds ================================
```
